README.md

```diff
@@ -125,36 +125,48 @@ When the user allows access you get a response back.
 
 This is the shape of the object that will be POSTed to the provided URL
 
-| Name      | Description                                 | Type           |
-| --------- | ------------------------------------------- | -------------- |
-| bloom_id  | The user's BloomID                          | `number`       |
-| token     | Unique string to identify this data request | `string`       |
-| signature | Request body signed by the Bloom app wallet | `string`       |
-| data      | Array of VerifiedData objects               | `VerifiedData` |
+| Name       | Description                                                             | Type           |
+| ---------- | ----------------------------------------------------------------------- | -------------- |
+| bloom_id   | The user's BloomID                                                      | `number`       |
+| token      | Unique string to identify this data request                             | `string`       |
+| signature  | Request body signed by the Bloom app wallet                             | `string`       |
```

Comment:
More specifically the signature is of the packedData string

Reply:
Changing this to match what's in the code
```
Signature of `packedData` by the user with their mnemonic.
```